duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -145,7 +144,7 @@ public Response executeHttpRequest() throws Exception {
     if (sigVersion == 1 || sigVersion == 2){
       requestBody = RequestBody.create(queryString, FORM_ENCODED);
     } else if (sigVersion == 5){
-      if (method == "POST" || method == "PUT"){
+      if (method.equals("POST") || method.equals("PUT")){
```

Comment:
quick java trick to avoid a NullPointerException if method ever ends up null somehow - flip the equals():
`if ("POST".equals(method) || "PUT".equals(method))`